cmake/openenclave-config.cmake.in

```diff
@@ -123,6 +123,18 @@ if(NOT TARGET openenclave::oegdb)
   set_target_properties(openenclave::oegdb PROPERTIES IMPORTED_LOCATION ${OE_BINDIR}/oegdb)
 endif ()
 
+# This target is an external project, so we have to manually
+# "export" it here for users of the package.
+if (UNIX)
```

Comment:
in a comment you said this was SGX only, and if so shouldn't this also check if SGX?

Reply:
I'd though that this file was SGX specific; but turns out it has an SGX check.

Moved the code to under an SGX check.
Also fixed oegdb target right above to be SGX only.